### PR TITLE
Prepare partners app for Python 3

### PR DESCRIPTION
**Pull request by dpoirier**

### CODE REVIEW

File: EquiTrack/partners/tests/test_views.py [hunk 1]
```diff
@@ -1489,7 +1490,7 @@ def test_intervention_list_minimal(self):
         )
 
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertEqual(response.data[0].keys(), ["id", "title"])
+        self.assertEqual(list(response.data[0].keys()), ["id", "title"])
```
Comment (reinbach) [inline EquiTrack/partners/tests/test_views.py]: Should this list be sorted?

File: EquiTrack/partners/serializers/exports/interventions.py [hunk 2]
```diff
@@ -385,7 +385,7 @@ def get_fr_numbers(self, obj):
         return ', '.join([x.fr_number for x in obj.frs.all()]) if obj.frs.all().count() > 0 else ""
 
     def get_migration_error_msg(self, obj):
-        return ', '.join([a for a in obj.metadata['error_msg']]) if 'error_msg' in obj.metadata.keys() else ''
+        return ', '.join(obj.metadata.get('error_msg', []))
```
Comment (reinbach) [inline EquiTrack/partners/serializers/exports/interventions.py]: I'm curious, why this change was made?

Comment (dpoirier) [reply EquiTrack/partners/serializers/exports/interventions.py]: I guess it's not really related to Python 3, but it took me a while to figure out what this line did and I thought simplifying it would save time for the next person who looked at it.

File: EquiTrack/partners/tests/test_views.py (hunk 1 above)
Comment (dpoirier) [reply EquiTrack/partners/tests/test_views.py]: Good idea.  I've updated this PR.

**Comment by ntrncic** [comment]
@dpoirier please resolve conflicts

**Comment by dpoirier** [comment]
@ntrncic conflicts resolved
